src/CosmosDB/CosmosDB.Test/CosmosDB.Test.csproj

```diff
@@ -5,7 +5,7 @@
   <Import Project="$(MSBuildThisFileDirectory)..\..\Az.Test.props" />
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.KeyVault" Version="3.0.1" />
-    <PackageReference Include="Microsoft.Azure.Management.CosmosDB" Version="3.9.1-preview" />
```

Comment:
this is a temp version from building locally from swagger. not the actual version